<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>338</width>
    <height>139</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QFileCombiner</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="directory_path_line_edit">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="directory_select_button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>SelectDirectory</string>
          </property>
          <property name="autoRepeat">
           <bool>false</bool>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
          <property name="autoDefault">
           <bool>true</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Keywords</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="keyword_line_edit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>.dat</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="combine_button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Combine!</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>directory_select_button</sender>
   <signal>pressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>SelectDirectory()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>324</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>combine_button</sender>
   <signal>pressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>Combine()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>288</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>keyword_line_edit</sender>
   <signal>returnPressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>Combine()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>148</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>45</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>SelectDirectory()</slot>
  <slot>Combine()</slot>
 </slots>
</ui>
